CHANGELOG.md

```diff
@@ -21,7 +21,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Upgrade CUDA library to version 12.2.0.
 - Upgrade NVIDIA Fabric manager to `nvidia-fabricmanager-535`
 - Increase EFS-utils watchdog poll interval to 10 seconds. Note: This change is meaningful only if [EncryptionInTransit](https://docs.aws.amazon.com/parallelcluster/latest/ug/SharedStorage-v3.html#yaml-SharedStorage-EfsSettings-EncryptionInTransit) is set to `true`, because watchdog does not run otherwise.
-- Upgrade EFA installer to `1.25.0`
+- Upgrade EFA installer to `1.25.1`
   - Efa-driver: `efa-2.5.0-1`
   - Efa-config: `efa-config-1.15-1`
   - Efa-profile: `efa-profile-1.5-1`
```

Comment:
The openmpi version should be upgraded to "openmpi40-aws-4.1.5-4"

Reply:
Thanks!